src/Dashboard/Data/RecentInvocationEntry.cs

```diff
@@ -16,6 +16,7 @@ public class RecentInvocationEntry
         private const string StartTimeKey = "StartTime";
         private const string EndTimeKey = "EndTime";
         private const string SucceededKey = "Succeeded";
+        private const string HeartbeatExpritedTimeKey = "Heartbeat2Name";
```

Comment:
Perhaps rename to just FunctionInstanceHeartbeatExpiry (removing the 2 and following the naming convention of the others)